_posts/2020-04-24-roboarmcontroller-intro.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "RoboArm Controller, part I: Intro and architecture."
```

Comment:
Titles should have consistent capitalization to improve readability. I usually use this tool for the job (choose and stick to one of the conventions throughout the blog): https://capitalizemytitle.com